### DISCUSSION TITLE
Auto tag 'mc_cid' links as coming from Mailchimp

### DISCUSSION THREAD

**Discussion by evolvedwebsites** [plausible/analytics#2876] [Anything Else]
Hi all, love your work and Plausible, which I now use exclusively instead of Google Analytics.

The manual link tagging is a great feature and works well.

One missing link tag however is for Mailchimp... 

All referred traffic shows it as 'Direct / None', it could be a fairly simple addition to include the 'mc_cid' as traffic from Mailchimp.
e.g.
all urls from Mailchimp emails end with something like this:
?mc_cid=8353584060&mc_eid=aa4e4f7225

Thanks for your consideration.

**Comment by metmarkosaric** [comment]
thanks for the kind words and the feedback @evolvedwebsites! we'll see the feedback we get and consider it in the future. currently to track these you could tag with utms instead or alongside mc_cid

**Reply by evolvedwebsites** [reply]
Thank you
